tests/cases/fourslash/codeFixPromiseToAsyncCatchAndRej.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path='fourslash.ts' />
+
+// @target: es6
+
+////function [|f|]():Promise<void> {
+////    return fetch('http://yahoo.com').then(result => { console.log(result); }, rejection => { console.log("rejected:", rejection); }).catch(err => { console.log(err) });
+////}
+
+verify.getSuggestionDiagnostics([{
+    message: "This may be converted to use async and await.",
+    code: 80006,
+}]);
+
+
+verify.codeFix({
+    description: "Convert to use async and await",
+    index: 0,
+    newFileContent:
+`async function f():Promise<void> {
+    try {
+        try {
+            var result = await fetch('http://yahoo.com');

```

Comment:
I think others have mentioned that using `var` and letting it leak out of this scope is a little sketchy.